### PR TITLE
feat(me): add support level

### PR DESCRIPTION
**Pull request by lizardK**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#947.0 ✘  
* 
  * build ✔ 
  * test ✘

File: src/me/me.v6.service.js [hunk 1]
```diff
@@ -23,7 +23,12 @@ angular.module("ovh-api-services").service("OvhApiMeV6", function ($resource, $c
             method: "GET",
             url: "/me/consumption/usage/history",
             isArray: true
-        }
+        },
+        supportLevel: {
+          method: "GET",
+          url: "/me/supportLevel",
+          isArray: false
```
Comment (FredericEspiau) [inline src/me/me.v6.service.js]: No need for this, `isArray` is `false` by default

Comment (antleblanc) [reply src/me/me.v6.service.js]: Explicit version has been preferred in that case :)

**Comment by ovh-cds** [comment]
CDS Report build#948.0 ✘  
* 
  * build ✔ 
  * test ✘

**Pull request merged**